src/legacy/core_plugins/vis_type_table/public/paginated_table/rows.js

```diff
@@ -21,6 +21,21 @@ import $ from 'jquery';
 import _ from 'lodash';
 import tableCellFilterHtml from './table_cell_filter.html';
 
+function getFormattedValue(formatter, value) {
+  let formattedValue;
+
+  if (formatter.htmlConvert) {
```

Comment:
Not sure that we should check for the `htmlConvert` class field here. It's an internal property which should be marked as protected.